string id
---
# result definition
bool success
string message
